fairlearn/metrics/_extra_metrics.py

```diff
@@ -6,38 +6,63 @@
 These are metrics which are not part of `scikit-learn`.
 """
 
+import numpy as np
 import sklearn.metrics as skm
 
 from ._balanced_root_mean_squared_error import _balanced_root_mean_squared_error  # noqa: F401
 from ._mean_predictions import mean_prediction, _mean_overprediction, _mean_underprediction  # noqa: F401,E501
 from ._selection_rate import selection_rate  # noqa: F401,E501
 
+_NEED_TWO_UNIQUE_Y_VALS = "Must have two unique y values"
+_NEED_POS_LABEL_IN_Y_VALS = "Must have pos_label in y values"
 
-def true_positive_rate(y_true, y_pred, sample_weight=None):
+
+def _get_labels_for_confusion_matrix(y_true, y_pred, pos_label):
+    """Figure out the labels argument for skm.confusion_matrix.
+
+    This assumes the input arrays are binary, and we need to specify a pos_label.
+    Given the way the calling routines work, this is achieved by ensuring that
+    the pos_label is last in the (two element) list.
+    """
+    my_labels = list(np.unique(np.concatenate((y_true, y_pred), axis=None)))
+    if len(my_labels) != 2:
+        raise ValueError(_NEED_TWO_UNIQUE_Y_VALS)
+    if pos_label not in my_labels:
+        raise ValueError(_NEED_TWO_UNIQUE_Y_VALS)
+    if my_labels[1] != pos_label:
+        my_labels = list(reversed(my_labels))
+    return my_labels
+
+
+def true_positive_rate(y_true, y_pred, sample_weight=None, pos_label=1):
```

Comment:
I think we could really use the docstrings here. As a user I'd be confused about `pos_label` because TPR has a definition that relies on labels being 0 and 1 and positives being the samples with `y_true=1`.

Reply:
Surely for true positive rate, you just need to know which label of the two is considered the 'positive' one?

Fair point about doing the docstrings, though.

Reply:
New style docstrings:
https://406-133444044-gh.circle-artifacts.com/0/docs/_build/html/api_reference/fairlearn.metrics.html#fairlearn.metrics.true_positive_rate
Before I go and write them all, let me know what you think